src/loader.js

```diff
@@ -107,7 +107,7 @@ export function pitch(request) {
         }
         this[NS](text, query);
         if (text.locals && typeof resultSource !== 'undefined') {
-          resultSource += `\nmodule.exports = ${JSON.stringify(text.locals)};`;
+          resultSource += `\nexport default ${JSON.stringify(text.locals)};`;
```

Comment:
```js
resultSource += `\nexport const locals = ${JSON.stringify(text.locals)};`;
```

Reply:
Needs triage which export is the best solution here

Reply:
What is the advantage of this over `export default`?

Reply:
Using default export means it works with all current CSS module imports (and hopefully avoid a major version bump?). With named export, we would have to change

```js
import app from './app.css'
import specific from './specific.sass'
```
to
```js
import { local as app } from './app.css'
import { local as specific } from './specific.sass'
```

Reply:
I agree with you, it's not that I'm in favor of a named export, but I need to revisit the `css-loader` first to check

>  (and hopefully avoid a major version bump?)

Unfortunately unlikely because of  `require('./file.css)` (CJS) usage 😥 

```
require('./file.css') => require('./file.css').default
```